server/controllers/bookController.js

```diff
@@ -134,6 +136,63 @@ class BookController {
       Utils.errorStatus(res, 500, error.message);
     }
   }
+
+  static async recieveBook(req, res){
+    const { isbn, patronId } = req.body;
```

Comment:
This funCtion name nearly messed with me, i feel returnBook would have been a better name.

Reply:
+1